docs/user-guide/how-to.md

```diff
@@ -186,6 +187,17 @@ permissions. To understand why an object is not being propagated to a namespace,
 use `kubectl hns describe <ns>`, where `<ns>` is either the source (ancestor) or
 destination (descendant) namespace.
 
+<a name="use-hrq"/>
+
+### Limit Resources over parent namespaces
+
+HNC has an object called `HirarchicalResourceQuota` that is similar to `ResourceQuota`
+but across all the namespaces in a hierarchy. It allows you to distribute your resources between
+teams, and those teams can distribute their resources between their subteams...
+[learn how it works](concepts.md#hierarchical-resource-quota) or see an [quickstart example](quickstart.md#hrq)
+
+Note: decimal point values cannot be specified in HRQ (you can't do `cpu: 1.5`). see [#292](https://github.com/kubernetes-sigs/hierarchical-namespaces/issues/292)
```

Comment:
see -> See (capitalized)

Maybe also say "You can't say `cpu: 1.5`, but you can say `cpu: 1500m`".